devdoc/platform_requirements.md

```diff
@@ -13,7 +13,7 @@ may find it makes more sense to perform these operations outside of the scope of
 In that case, the `platform_init` and `platform_deinit` calls may be left empty.
 
 ### References 
-[Azure IoT porting guide](https://github.com/Azure/azure-c-shared-utility/blob/master/doc/porting_guide.md)<br/>
+[Azure IoT porting guide]https://github.com/Azure/azure-c-shared-utility/blob/master/devdoc/porting_guide.md)<br/>
```

Comment:
Missing a parenthesis here which is cause the link to include the <br/> at the end of the line.